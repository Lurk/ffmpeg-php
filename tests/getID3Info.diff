002- ffmpeg getArtist(): Alexander Thomas
003- ffmpeg getAlbum(): MP3 test files (http://www.dr-lex.be/software/testsounds.html#Tones)
004- ffmpeg getGenre(): Sound Clip
005- ffmpeg getTrackNumber(): 3
006- ffmpeg getYear(): 2008