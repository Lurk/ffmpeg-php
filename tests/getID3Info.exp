ffmpeg getTitle(): Test mp3
ffmpeg getArtist(): Alexander Thomas
ffmpeg getAlbum(): MP3 test files (http://www.dr-lex.be/software/testsounds.html#Tones)
ffmpeg getGenre(): Sound Clip
ffmpeg getTrackNumber(): 3
ffmpeg getYear(): 2008