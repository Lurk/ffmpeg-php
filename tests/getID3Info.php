<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/16Hz-20kHz-Exp-1f-5sec.mp3');
printf("ffmpeg getTitle(): %s\n", $mov->getTitle());
printf("ffmpeg getArtist(): %s\n", $mov->getArtist());
printf("ffmpeg getAlbum(): %s\n", $mov->getAlbum());
printf("ffmpeg getGenre(): %s\n", $mov->getGenre());
printf("ffmpeg getTrackNumber(): %s\n", $mov->getTrackNumber());
printf("ffmpeg getYear(): %s\n", $mov->getYear());

?>
