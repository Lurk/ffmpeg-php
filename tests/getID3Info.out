ffmpeg getTitle(): Test mp3